Source/Core/Core/AchievementManager.h

```diff
@@ -9,7 +9,6 @@
 #include <functional>
 #include <map>
 #include <mutex>
-#include <string>
```

Comment:
Pretty sure you need to keep this. While you're here, this header seems to be missing several includes:

- `memory`
- `set`
- `string_view`
- `vector`
- `Common/CommonTypes.h`

Reply:
Did I accidentally press Ctrl+X instead of Ctrl+C on this or something 🤨

I've added all the ones you mentioned, plus `chrono`.